gammapy/cube/tests/test_basic_cube.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+import numpy as np
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from ...maps import MapAxis, WcsGeom, WcsNDMap
+from ...data import DataStore
+from ...cube.basic_cube import make_map_counts
+
+# initial time of run 110380 from cta 1DC
+t0_1DC = 664504199. * u.s
+
+energy_true = np.logspace(-2., 2., 30) * u.TeV
+energy_reco = np.logspace(-1., 1.5, 10) * u.TeV
+times = np.linspace(t0_1DC.value, t0_1DC.value+1900., 10)*u.s
+
+axis_energy_true = MapAxis(energy_true.value, interp='log', node_type='edge',
+                           name='energy_true', unit = energy_true.unit)
+axis_energy_reco = MapAxis(energy_reco.value, interp='log', node_type='edge',
+                           name='energy_reco', unit = energy_reco.unit)
+axis_time = MapAxis(times, node_type='edge', name='time',unit = times.unit)
+
+cta_1dc_store = "$GAMMAPY_EXTRA/datasets/cta-1dc/index/gps/"
+cta_1dc_runs = [110380]
+
+hess_crab_store = "$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/"
+hess_crab_runs = [23523]
```

Comment:
Can we avoid loading events from disk?
If we have a little `make_event_list` example function (or pytest fixture), it's faster, and the tests become better because we control the events that go in, i.e. can even choose them so that we know which bins should be filled.

Reply:
Well building an EventList from scratch is possible but tedious. Is it really a strong limitation?

Reply:
If [little Peter](https://github.com/gammapy/gammapy/issues/1304#issuecomment-365357927) can do it, we can with ~ 10 lines. :-)

But yes, this can go in as-is since we're short on time. I would suggest you leave a comment like this:
```
# TODO: change the test event list to something that's created from scratch,
# using values so that it's possible to make simple assert statements on the
# map data in the tests below, i.e. have pixels that should receive 0, 1 or 2 counts
```
Having those tests really is an advantage, they catch errors concerning coords / map interface, like off by one pixel, units, wrong axis, wrong index, ...